### PR TITLE
long names

### PR DESCRIPTION
**Pull request by apprb**
Long name in filetransfer widget: fixed
Word wrap in the chat form: time stamps: OK, names: partial

### CODE REVIEW

File: filetransferinstance.cpp [hunk 1]
```diff
@@ -34,6 +34,12 @@ FileTransferInstance::FileTransferInstance(ToxFile File)
     remotePaused = false;
 
     filename = File.fileName;
+
+    QFont font("sans");
```
Comment (tux3) [inline filetransferinstance.cpp:38]: What if "sans" isn't available ? What if it's available but not what you expect it to be ? 

Is this just supposed to load Helvetica ? 
http://qt-project.org/doc/qt-4.8/qfont.html#fontmatching

File: filetransferinstance.cpp [hunk 2]
```diff
@@ -41,9 +47,12 @@ FileTransferInstance::FileTransferInstance(ToxFile File)
     {
         QImage preview;
         File.file->seek(0);
-        if (preview.loadFromData(File.file->readAll()))
+        if (File.file->size() <= 1024*1024*25)
```
Comment (tux3) [inline filetransferinstance.cpp:50]: Well, that's not a bad idea, but I'd appreciate if you mentioned it in the PR description or even just in the commit message. That's completely unrelated to long names and fixing #324.

Comment (apprb) [inline filetransferinstance.cpp:50]: My bad. Completely forgot that I've done this fix too.

File: filetransferinstance.cpp (hunk 1 above)
Comment (apprb) [inline filetransferinstance.cpp:38]: I looked through code and it looks like just `Qfont font` is enough (for example: addfriendform.cpp line 32).

**Comment by apprb** [comment]
So it is better to make this PR again than try to correct this one.

**Pull request closed without merging**